src/domain/Phone/WebRTCPhone.js

```diff
@@ -22,19 +22,27 @@ export default class WebRTCPhone extends Emitter implements Phone {
 
   audioOutputDeviceId: string | typeof undefined;
 
+  audioRingDeviceId: string | typeof undefined;
+
   ringingEnabled: boolean;
 
   acceptedSessions: Object;
 
   rejectedSessions: Object;
 
-  constructor(client: WazoWebRTCClient, audioOutputDeviceId?: string, allowVideo: boolean = false) {
+  constructor(
+    client: WazoWebRTCClient,
+    audioOutputDeviceId?: string,
+    audioRingDeviceId?: string,
+    allowVideo: boolean = false,
+  ) {
     super();
 
     this.client = client;
     this.allowVideo = allowVideo;
     this.sipSessions = {};
     this.audioOutputDeviceId = audioOutputDeviceId;
+    this.audioRingDeviceId = audioRingDeviceId;
```

Comment:
Can you set `audioRingDeviceId` to `audioOutputDeviceId` if not defined ? (again BC break ...)

Reply:
geeeez... good catch, can't believe i let this one slide